### PR TITLE
Simplify date formatting for the Account Console (v3)

### LINKED ISSUE #21660: Wrong convert timestamp to date
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

account/ui

### Describe the bug

Error when converting timestamp to date. 

Снимок экрана 2023-07-12 в 17 59 13

Снимок экрана 2023-07-12 в 18 00 05

### Version

21.1.2, 22.0.0

### Expected behavior

When converting timestamp 1689173915, the date is January 20, 1970

### Actual behavior

When converting timestamp 1689173915, the date should be June 23, 2023

### How to Reproduce?

This problem is displayed on the active devices page

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by jonkoops**
Simplifies the date formatting logic for the Account Console (v3), by placing all the date formatting logic in a single re-usable utility function. This also fixes a bug where some of the dates were formatted incorrectly (see #21660). This also fixes an issue where the date was formatted in a US English format, even when the user has a different locale configured.

Closes #21660

### CODE REVIEW

File: js/apps/account-ui/src/applications/Applications.tsx [hunk 1]
```diff
@@ -227,14 +228,7 @@ const Applications = () => {
...
                       <DescriptionListDescription>
-                        {new Intl.DateTimeFormat("en", {
-                          year: "numeric",
-                          month: "long",
-                          day: "numeric",
-                          hour: "numeric",
-                          minute: "numeric",
-                          second: "numeric",
-                        }).format(application.consent.createdDate)}
+                        {formatDate(new Date(application.consent.createdDate))}
```
Comment (jonkoops) [inline js/apps/account-ui/src/applications/Applications.tsx:231]: @edewit I am not quite sure how to get this piece of UI to show up. Could you test this, or tell me how to test it? I need to know if we need to multiply this value by a 1000, like in the other places where this bug was popping up.

Comment (edewit) [reply js/apps/account-ui/src/applications/Applications.tsx:231]: you need to enable consent on the client:
image
Then you get it, but it seems that we are missing some translations:
image

**Review by edewit — APPROVED** [review]
looking good

File: js/apps/account-ui/src/applications/Applications.tsx (hunk 1 above)
Comment (jonkoops) [reply js/apps/account-ui/src/applications/Applications.tsx:231]: We should certainly fix that one, can you log an issue for it?

**Pull request merged**